components/domains/registerCheckboxes.tsx

```diff
@@ -21,7 +23,13 @@ const RegisterCheckboxes: FunctionComponent<RegisterCheckboxes> = ({
           className="flex items-center justify-left text-xs mr-2 cursor-pointer"
           onClick={onChangeTermsBox}
         >
-          <Checkbox checked={termsBox} sx={{ padding: 0 }} />
+          <Checkbox
+            checked={termsBox}
+            sx={{
+              padding: 0,
+              background: variant === "white" ? "#FFF" : "transparent",
```

Comment:
I think we can do that as a CSS property here is an example https://github.com/starknet-id/app.starknet.id/pull/433/files#diff-f0c5d9230652694fe4a3a688f468f7cb68e87ba80c2bb944fb4589181ec144a7

Reply:
you mean using `className` instead of having the css properties directly in `sx` right ?